jwst/cube_build/blot_cube_build.py

```diff
@@ -46,10 +44,10 @@ def __init__(self, median_model, input_models):
         self.detector = median_model.meta.instrument.detector
 
         # information on how the IFUCube was constructed
-        self.weight_power = median_model.meta.ifu.weight_power
-        self.weighting = median_model.meta.ifu.weighting
+        # self.weight_power = median_model.meta.ifu.weight_power
```

Comment:
If these lines are no longer of relevance, they should be removed completely. If there is a reason to keep them around but commented, having comments regarding to why they are commented but why they may still be of relevance.